### ISSUE TITLE
aria-hidden=true for "scroller"

### ISSUE THREAD

**Issue by stevenvachon** [tbranyen/hyperlist#52]
It's useless to accessibility and should be completely ignored. This can currently be achieved with the `scroller` option, but it should be default behaviour.
